internal/cli/atlas/maintenance/update.go

```diff
@@ -0,0 +1,90 @@
+// Copyright 2020 MongoDB Inc
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package maintenance
+
+import (
+	"github.com/mongodb/mongocli/internal/cli"
+	"github.com/mongodb/mongocli/internal/config"
+	"github.com/mongodb/mongocli/internal/flag"
+	"github.com/mongodb/mongocli/internal/store"
+	"github.com/mongodb/mongocli/internal/usage"
+	"github.com/spf13/cobra"
+	atlas "go.mongodb.org/atlas/mongodbatlas"
+)
+
+type UpdateOpts struct {
+	cli.GlobalOpts
+	cli.OutputOpts
+	dayOfWeek int
+	hourOfDay int
+	startASAP bool
+	store     store.MaintenanceWindowUpdater
+}
+
+func (opts *UpdateOpts) initStore() error {
+	var err error
+	opts.store, err = store.New(config.Default())
+	return err
+}
+
+var updateTemplate = "Maintenance window updated.\n"
+
+func (opts *UpdateOpts) Run() error {
+	err := opts.store.UpdateMaintenanceWindow(opts.ConfigProjectID(), opts.newMaintenanceWindow())
+	if err != nil {
+		return err
+	}
+	opts.Print(updateTemplate)
+	return nil
```

Comment:
[nit] ignored error, also you can pass nil if update doesn't return anything
```suggestion
	return opts.Print(nil)
```